### PR TITLE
Fix clusterIP assignment

### LINKED ISSUE #6307: Helm ClusterIP is None
## Expected Behavior

Minio to come up with a valid and accessible ClusterIP when installed in stand-alone mode with the helm chart with serviceType ClusterIP (default)

## Current Behavior

Unreachable - DNS not found from Kubernetes

## Possible Solution

I may be doing something wrong, @harshavardhana can you advise?

## Steps to Reproduce (for bugs)

Store the secets in Kubernetes

```
kubectl create secret generic -n openfaas-fn \
 s3-secret-key --from-literal s3-secret-key="$SECRET_KEY"
kubectl create secret generic -n openfaas-fn \
 s3-access-key --from-literal s3-access-key="$ACCESS_KEY"
```

Install Minio with helm

```
helm install --name cloud --namespace openfaas \
   --set accessKey=$ACCESS_KEY,secretKey=$SECRET_KEY,replicas=1,persistence.enabled=false,service.port=9000 \
  stable/minio
```

Helm told me that the name value should be `cloud-minio-svc.openfaas.svc.cluster.local`

### PR DESCRIPTION
**Pull request by nitisht**
**What this PR does / why we need it**: Fixes clusterIP assignment. 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes https://github.com/minio/minio/issues/6307

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by nitisht** [comment]
ping @scottrigby @unguiculus

File: stable/minio/values.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ persistence:
 
 service:
   type: ClusterIP
-  clusterIP: None
+  clusterIP:
```
Comment (paulczar) [inline stable/minio/values.yaml]: Can you set as `clusterIP: ~`   ( ~ is the [canonical](http://yaml.org/type/null.html) way to specify a null resource and is less confusing than an empty value ).

**Review by paulczar — CHANGES_REQUESTED** [review]

File: stable/minio/values.yaml (hunk 1 above)
Comment (nitisht) [reply stable/minio/values.yaml]: Done @paulczar . Thanks for the review

**Review by paulczar — APPROVED** [review]

**Pull request merged**